docs/apache-airflow/tutorial_taskflow_api.rst

```diff
@@ -266,6 +266,20 @@ Python version to run your function.
 These two options should allow for far greater flexibility for users who wish to keep their workflows more simple
 and Pythonic.
 
+Using the TaskFlow API with Sensor operators
+--------------------------------------------
+You can apply the ``@task.sensor`` decorator to convert a regular Python function to an instance of the
+BaseSensorOperator class. The Python function implements the poke logic and returns an instance of
+the ``PokeReturnValue`` class as the ``poke()`` method in the BaseSensorOperator does. The ``PokeReturnValue`` is
+a new feature in Airflow 2.3 that allows a sensor operator to push an XCom value as described in
+`Community Providers <https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers/howto/create-update-providers.rst>`_ under the section "Having sensors return XOM values".
```

Comment:
Don’t do this

Reply:
Can you be specific?

Reply:
Why are you linking to GitHub?

Reply:
I see. This is probably why breeze build-docs fails in the CI.